api/dist/kernel.json

```diff
@@ -551,7 +554,9 @@
       "UpdatePolicy": {
         "AutoScalingRollingUpdate": {
           "MaxBatchSize": 1,
-          "MinInstancesInService": 1
+          "MinInstancesInService": 1,
+          "PauseTime" : "PT15M",
```

Comment:
Yes that's just a timeout for receiving *all* the signals. We could even bump it to 20 or 30 

> On Oct 16, 2015, at 8:13 AM, David Dollar <notifications@github.com> wrote:
> 
> In api/dist/kernel.json:
> 
> > @@ -551,7 +554,9 @@
> >        "UpdatePolicy": {
> >          "AutoScalingRollingUpdate": {
> >            "MaxBatchSize": 1,
> > -          "MinInstancesInService": 1
> > +          "MinInstancesInService": 1,
> > +          "PauseTime" : "PT15M",
> Does it roll faster than this if it receives the signal?
> 
> —
> Reply to this email directly or view it on GitHub.
>